testutils/src/main/java/de/cyface/testutils/SharedTestUtils.java

```diff
@@ -191,7 +191,7 @@ public static int clearPersistenceLayer(@NonNull final Context context, @NonNull
         // Remove database entries
         final int removedGeoLocations = resolver.delete(getGeoLocationsUri(authority), null, null);
         final int removedMeasurements = resolver.delete(getMeasurementUri(authority), null, null);
-        // TODO: why does this break the life-cycle tests in DataCapturingServiceTest? - can't find an answer ...
+        // Unclear why this breaks the life-cycle tests in DataCapturingServiceTest.
```

Comment:
No need to further investigate this as this does not effect non-test environments